extensions/amp-analytics/0.1/test/test-events.js

```diff
@@ -1807,23 +1803,24 @@ describes.realWin('Events', {amp: 1}, env => {
           /* createReportReadyPromiseFunc */ null,
           saveCallback
         )
-        .returns(unlisten)
         .once();
-      tracker.add(analyticsElement, 'visible', config, eventResolver);
-      yield macroTask();
+      const res = tracker.add(
+        analyticsElement,
+        'visible',
+        config,
+        eventResolver
+      );
+      expect(res).to.be.a('function');
```

Comment:
I'll say checking the return value of `add` is a function is still valuable because it prevent cases where one fail to return an unlistener.